.github/workflows/build-docker.yml

```diff
@@ -0,0 +1,38 @@
+name: Build Docker
+
+# Controls when the action will run. 
+# Triggers the workflow for a pull request for
+# dev when the Dockerfile file changes.
+on:
+  pull_request:
+    branches: [ dev ]
+    paths: [ Dockerfile ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build_docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```

Comment:
looks weird formatting.

Reply:
I thought so too, but this action page https://github.com/docker/build-push-action had the same format. I can update it though, thanks for the review!